### ISSUE TITLE
Expanding the DWYL Trade Mark to US

### ISSUE THREAD

**Issue by markwilliamfirth** [dwyl/hq#228]
It will cost approximately £871.65 to expand the DWYL trade mark to the US.

Should we do this?

**Comment by nelsonic** [comment]
@markwilliamfirth I think this decision is for @iteles to make. 
I think an IP Lawyer should be consulted to get a clear understanding of what we are applying for, what we can expect to get and how long it will take...
Is it a _separate_ application for the `dwyl` name vs. the logo or can they be done simultaneously? (_preferably at a reduced cost_).

> My "_vote_" would be to _invest_ in the trademark because it _adds_ to the Twitter claim and will help us with IG too. Also, once it's done, it's done. and we can move on with building the product without "_fear_" ... 

I would `.apply()` for the trademark in US `before` the [request for domain transfer](https://github.com/dwyl/hq/issues/119) just so the ball is rolling with the USPTO but I would ***NOT*** _mention_ it in the _initial_ email to the current "_owner_" of dwyl.com ... 

if the chap who _currently_ has dwyl.com flat out refuses to negotiate we can `wait` for the trademark to come through and then `try` again, _politely_ reminding him that we have the trademark and would prefer to _pay_ him for all the money he has spent renting the domain until now rather than open a dispute with ICANN ... https://www.icann.org/resources/pages/dispute-resolution-2012-02-25-en#trademark

**Comment by iteles** [comment]
I'd go with speaking to an IP lawyer first to understand what it is we'd get for this and then also looking into #229 - if we're going to do this, we want to do it before the end of the financial year.

**Comment by markwilliamfirth** [comment]
@iteles noticed https://github.com/dwyl/intellectual-property/issues/3

Weird that London IP advised you that it would be 2k? Was that only for the word mark, "DWYL"?

Were there any questions they didn't answer? It seems pretty simple to me and London IP covered me asking about the states last time I called.

**Comment by markwilliamfirth** [comment]
@nelsonic I thought the point of the application was to mark `DWYL` so we could challenge the `.com` domain, do we need the logo too? Don't see much point to marking the logo at this moment in time.

**Comment by markwilliamfirth** [comment]
@nelsonic I spoke to an IP lawyer today and they said that we don't need to have a US trademark to claim under UDRP (have you heard otherwise?). Cost to file a UDRP dispute is $1500, equivalent £1202.98.

I suggest we email John and make an offer of approx $250 and go from there.

Alternatively we could wait until the expiration date in June (2017-06-15) to see if he fails to renew (unlikely to fail to renew in my opinion).

I'm in favour of moving fast and contacting John and making an offer.

**Comment by nelsonic** [comment]
@markwilliamfirth agree. also in _favour_ of "_moving fast_" and sending him an Email. please draft it in a Google Doc and share with @iteles. thanks 👍

**Comment by nelsonic** [comment]
@markwilliamfirth I think we should trademark in the US _independently_ of the email for the `.com`
just cause I _suspect_ that US will be a strong "_market_" for dwyl products & services and it would be good to have that be _unambiguous_ so we don't get some _random_ IP-Troll telling us we have to re-brand because they "own" dwyl in US...

**Comment by markwilliamfirth** [comment]
Putting this on hold until the £871.65 becomes worth it.

**Comment by nelsonic** [comment]
It's worth it _now_ (IMO). 
The sooner we get this ball rolling the sooner we can "_protect_" the "_brand_" ... 🥇 
the way I think about this kind of stuff is by considering the cost to _change_ brands for the US "_market_" if someone `else` registers the trademark (_the way "WeWork" did..._)

**Comment by iteles** [comment]
Let's get this done 👍 

@markwilliamfirth Could you spend 10 minutes looking at the paperwork and let me know how much time you think it will require so that we can determine whether #321 is a still top dog in terms of priorities?

**Comment by markwilliamfirth** [comment]
http://www.wipo.int/madrid/en/forms/

[form_mm2.pdf](https://github.com/dwyl/hq/files/1317571/form_mm2.pdf)

@iteles I have no idea how long it will take but I think this is the correct form

It should cost 1041 swiss francs (currently approx 799.95 British Pound)

**Comment by iteles** [comment]
Oh, sorry, I thought you were still going to provide an estimate for this to determine priority.
Yes please, let's!

I've reviewed the form and it doesn't look like any of the questions are 'essay' questions, they're mostly one liners and tick boxes so I'm thinking it won't take long if you're not interrupted (the `AMOUNT OF FEES` will take a little longer maybe).  

If you need an info from me just let me know, I'll keep an eye out for this issue.
